nodl_to_policy/_common/lifecycle_node.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Since we aren't including functionality for the lifecycle_node yet, should we just remove this file completely?

Reply:
I believe we should. I wasn't sure about the timeline of NoDL design discussion, so I left it in for now.

Reply:
Here's my suggestion.  We take the lifecycle_node.xml out of this PR (and the Enum).  Then we merge this PR.  Then we open another PR adding the lifecycle support back as a draft.  That way the code exists, but we don't have essentially "dead" code hanging around.

What do you think?

Reply:
Removed in aa269e5.

Reply:
> Here's my suggestion. We take the lifecycle_node.xml out of this PR (and the Enum). Then we merge this PR. Then we open another PR adding the lifecycle support back as a draft. That way the code exists, but we don't have essentially "dead" code hanging around.

Yes, I hadn't thought of this option. It's better than leaving the Enum around like in aa269e5. I will remove all of it from this PR and open a draft later then.

Reply:
Removed both lifecycle_node.xml and the Enum. Will open a draft PR related to this topic.